Engine/EditorUI/Panel/InspectorSubpanel/PanelComponent.cpp

```diff
@@ -1372,9 +1381,26 @@ void PanelComponent::ShowComponentMeshColliderWindow(ComponentMeshCollider* mesh
 		}
 	}
 }
+
+void PanelComponent::ShowComponentAudioListenerWindow(ComponentAudioListener* component_audio_listener)
+{
+	if (ImGui::CollapsingHeader(ICON_FA_HEADPHONES " Audio Listener", ImGuiTreeNodeFlags_DefaultOpen))
+	{
+		if (!ShowCommonComponentWindow(component_audio_listener))
+		{
+			return;
+		}
+		ImGui::Separator();
+
+		//Add More Stuff here 
```

Comment:
Do we need more stuff here?

Reply:
I have in mind having the Pause/Resume button options for the audio here